ci.sh

```diff
@@ -48,7 +48,20 @@ cd UBCRocketGroundStation
 
 # Unit tests & integration tests
 source venv/bin/activate
-python -m pytest
+mkdir test_reports
+
+# Unit
+coverage run --omit 'venv/*' -m pytest --junitxml=test_reports/junit.xml --ignore=tests/integration_tests --ignore-glob='tests/*_sim.py'
+coverage report --omit 'venv/*'
+coverage xml -o test_reports/unit-test-coverage.xml
+head test_reports/unit-test-coverage.xml
+
+# Integration
+coverage run --omit 'venv/*' -m pytest --junitxml=test_reports/junit.xml tests/integration_tests tests/*_sim.py
```

Comment:
Does this mean that files like `tests/test_hw_sim.py` are being included in the integration test report. Was there a reason for that? That file is not actually an integration test, I know, confusing naming.

It might be cleaner to make a `tests/unit_tests` folder and you can just put everything that's not in the `integration_tests` folder in there.